Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -207,8 +208,16 @@ public static void ApplyChanges(this InterceptorPool pool, IEnumerable<ITrackabl
                 {
                     // Mark modified properties
                     SetEntityState(context, item, parent, propertyName, EntityState.Unchanged, interceptors);
-                    foreach (var property in item.ModifiedProperties)
-                        context.Entry(item).Property(property).IsModified = true;
+                    var entry = context.Entry(item);
+                    // Exclude properties attributed as NotMapped
+                    var notMappedProps = item.GetType()
```

Comment:
OK, so see if you can make this EDM based.  You need to write a private method called `IsMappedProperty(string propertyName)` and call it like so:

```csharp
// Set modified properties
if (item.TrackingState == TrackingState.Modified
    && (state == null || state == TrackingState.Modified)
    && item.ModifiedProperties != null
    && item.ModifiedProperties.Count > 0
    && !IsMappedProperty(propertyName)
)
```

Put it in the `ApplyChanges Helpers` region of `DbContextExtensions`.  You can make use of the `GetEdmSpaceType` method.  I think the `DeclaredProperties` of `EntityType` should exclude the `NonMapped` (data annotations) and `Ignored` (fluent API) properties, but I'm not sure.

Hopefully this helps! :)